server/src/http/feature_refresher.rs

```diff
@@ -0,0 +1,1593 @@
+use std::collections::HashSet;
```

Comment:
Moving this to a new file in this PR makes this really challenging to review. I'm guessing some stuff has changed here?

Reply:
TBH, I wasn't aware I did.. where was it located earlier?

Reply:
Turns out dx had moved it to the refresher module, and my branch hadn't caught that. Removed my copy and kept the one inside the refresher module.